src/Compilers/Core/Portable/CodeAnalysisResources.resx

```diff
@@ -453,7 +453,10 @@
     <value>Invalid data at offset {0}: {1}{2}*{3}{4}</value>
   </data>
   <data name="SymWriterNotDeterministic" xml:space="preserve">
-    <value>SymWriter doesn't support deterministic compilation</value>
+    <value>Windows PDB writer doesn't support deterministic compilation (is the compiler host missing Microsoft.DiaSymReader.Native library?)</value>
+  </data>
+  <data name="SymWriterNotAvailable" xml:space="preserve">
+    <value>Windows PDB writer is not available (is the compiler host missing Microsoft.DiaSymReader.Native library?)</value>
```

Comment:
Maybe just `Windows PDB writer is not available -- could not find Microsoft.DiaSymReader.Native`?